Formula/gitwatch.rb

```diff
@@ -0,0 +1,19 @@
+class Gitwatch < Formula
+  desc "Watch a file or folder and automatically commit changes to a git repo easily"
+  homepage "https://github.com/gitwatch/gitwatch"
+  url "https://github.com/gitwatch/gitwatch/archive/refs/tags/v0.1.tar.gz"
+  sha256 "ed52d5c799c19dc6f920f3625964bc4b4948b6f8929c289aece462304e419697"
+  license "GPL-3.0-or-later"
+  head "https://github.com/gitwatch/gitwatch.git"
+  bottle :unneeded
+  depends_on "coreutils"
+  depends_on "fswatch"
+
+  def install
+    bin.install "gitwatch.sh" => "gitwatch"
+  end
+
+  test do
+    system "gitwatch"
```

Comment:
Can we have a more substantial test that tests the actual functionality of the program?

Reply:
I thought about this a bit but it would involve creating a new git repository, launching gitwatch, creating a file, waiting some time until fswatch picks up the file change, and then testing if there is a git commit. I think it can be done but it I'm afraid it could be quite brittle. Also, I don't know if manipulating a git repository inside an automatic build environment would cause any issues.

Reply:
Working with a repository should be fine, we have other formulae initialising/manipulating repositories in the `test` block.

> creating a new git repository, launching gitwatch, creating a file, waiting some time until fswatch picks up the file change, and then testing if there is a git commit.

We should be able to do this (you can use `sleep` for the waiting part).

Reply:
Ok, I'll give it a try.

Reply:
I've added a test.

Reply:
Just saw that the test fails brew audit. I'll push an update soon.